Library/Formula/slimerjs.rb

```diff
@@ -1,3 +1,23 @@
+class FirefoxDependency < Requirement
+  fatal true
+  cask "Firefox"
+  download "https://www.mozilla.org/en-US/firefox/new"
+
+  satisfy { firefoxappdira? || firefoxappdirb? }
+
+  def firefoxappdira?
+    File.exist?("~/Applications/Firefox.app")
+  end
```

Comment:
As far as I remember `File.exists?` does not catch symlinks. Either use File.symlink? or something like `brew cast list | grep firefox`